pkg/reconciler/revision/resources/pa.go

```diff
@@ -36,8 +37,16 @@ func MakePA(rev *v1alpha1.Revision) *av1alpha1.PodAutoscaler {
 			Namespace: rev.Namespace,
 			Labels:    makeLabels(rev),
 			Annotations: resources.FilterMap(rev.GetAnnotations(), func(k string) bool {
-				// Ignore last pinned annotation.
-				return k == serving.RevisionLastPinnedAnnotationKey
```

Comment:
I usually like switches, but here it feels
`return a ==b || c==d&&!routable`
is easier to read, especially that the code is inline.